src/Indice.Features.Cases.AspNetCore/Models/GetCasesListFilter.cs

```diff
@@ -9,11 +9,10 @@ namespace Indice.Features.Cases.Models;
 /// </summary>
 public class GetCasesListFilter
 {
-    /// <summary>The Id of the customer to filter.</summary>
-    public string CustomerId { get; set; }
-
-    /// <summary>The name of the customer to filter.</summary>
-    public string CustomerName { get; set; }
+    /// <summary>
+    /// A collection of properties to filter the cases by. It's list of FilterClause
+    /// </summary>
+    public List<FilterClause> CaseListData { get; set; } = new List<FilterClause>();
```

Comment:
Δεν μου φαινεται τοσο κατατοπιστικό, ίσως CaseListFilter? To data νιωθω με παραπεμπει σε case data

Reply:
αν ειναι too much, το αλλάζουμε/σκεφτόιμαστε σε μελλοντικό PR που θα προσθέσουμε και τα υπόλοιπα

Reply:
Θα του αλλάξω όνομα, δεν είναι κάτι.

Reply:
CasesListFilterClauses? Αλλά μετά μπορεί να πει κάποιος πως "και τα metadata είναι List<FilterClause> αλλά τα ονομάσατε metadata..."